src/components/StarRating/StarRating.jsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+import * as S from './StarRating.styles';
+
+class StarRating extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stars: [],
+      rating: 0,
+      hovered: 0,
+      selectedIcon: '★',
+      deselectedIcon: '☆'
```

Comment:
why do you need to store icons in state?

Reply:
Fixed, I created separate file for icons.